internal/shim-sev/src/syscall.rs

```diff
@@ -62,7 +65,6 @@ pub unsafe fn _syscall_enter() -> ! {
     push   rdi
     push   rsi
     push   rdx
-    push   rcx
```

Comment:
You are totally correct!
See https://github.com/enarx/enarx-keepldr/commit/67a22fc51154ba539e9d27f2f003359f7c42cedf
I shall make a big fat comment to prevent removal again, only because I think it is clobbered anyway.

Reply:
Fixed with https://github.com/enarx/enarx-keepldr/pull/166